Packages/doc/labnotebook-docs.rst

```diff
@@ -4,18 +4,28 @@ Labnotebook documentation for developers
 Overview
 ========
 
-In MIES we store all metadata of the sweeps and the testpulse in a data
-structure we call the "labnotebook". The general handling
-is like an old-school paper lab journal where each entry has a date and time and
-nothing is ever erased. Our digital version consists of four multi dimensional
-arrays, one pair of `key`/`value` arrays for numerical data and another for
-textual data.
-
-Assuming you read the data from NWB files these arrays are located in
-`/general/labnotebook/$device` and are called `numericalKeys`,
-`numericalValues`, `textualKeys` and `textualValues`. Each pair, the `key` and
-`value` array, is used to describe all metadata entries and also hold the value
-of each entry.
+In MIES, metadata of sweeps and test pulses get stored in a data structure
+called a *labnotebook*. The general structure is like a paper-based lab journal
+with columnar data entry storage. Each measurement has an associated date and
+time of the data acquisition and any further modifications. After data
```

Comment:
I'm not sure what `modifications` are. Perhaps provide an example?

Reply:
@ukos-git Nice enhancement. I do second the question regarding `modification`.

Reply:
I was thinking of the nwb file format, where the modification date is also present. Obviously a mistake. I restored the original formulation.